course_discovery/apps/course_metadata/models.py

```diff
@@ -1275,6 +1276,10 @@ class CourseRun(DraftModelMixin, CachedMixin, TimeStampedModel):
 
     invite_only = models.BooleanField(default=False)
     featured = models.BooleanField(default=False)
+    marketing_price = models.BooleanField(default=False, verbose_name=_('Price'), help_text=_(
```

Comment:
why is it named marketing price but it is a boolean? seems like it would be a float value. This field should be renamed like `is_marketing_price` etc.

Reply:
The field name are same which are set in the WordPress

Reply:
the field name on the wordpress seems to be `price`. adding `marketing_` prefix makes it confusing. If we are changing it even this much, I think we should change it to make logical.

Reply:
@rehan99000 we have updated this field name to `is_marketing_price_set`
https://github.com/edly-io/course-discovery/pull/17#discussion_r622788202